homeassistant/components/emulated_hue/hue_api.py

```diff
@@ -517,21 +517,35 @@ def get_entity_state(config, entity):
 
 def entity_to_json(config, entity, state):
     """Convert an entity to its Hue bridge JSON representation."""
+    entity_features = entity.attributes.get(ATTR_SUPPORTED_FEATURES, 0)
+    if entity_features:
```

Comment:
supported features is a bitmap,  you need to compare it with `SUPPORT_BRIGHTNESS` to see if it supports brightness.

Reply:
This will also need a test added.

Reply:
I am not a good programmer can somebody help me to add the test?

Reply:
Use this test as a template. 

https://github.com/home-assistant/home-assistant/blob/dev/tests/components/emulated_hue/test_hue_api.py#L223

Reply:
how I can create a new lamp, which doesn't support brightness for the test?

Reply:
`hass.states.async_set('light.no_brightness', 'on', {})`